torchmultimodal/modules/encoders/clip_text_encoder.py

```diff
@@ -87,7 +87,7 @@ def initialize_parameters(self):
 
     def build_attention_mask(self):
```

Comment:
no return annotation needed?

Reply:
It is probably due to [this property](https://mypy.readthedocs.io/en/stable/common_issues.html#no-errors-reported-for-obviously-wrong-code) of mypy where functions without any annotations are ignored completely. Not sure why this is, but we should see if there is some solution for this behavior.